src/main/resources/db/migration/V1__creating_tables.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE Employee (
+    employee_id INT AUTO_INCREMENT,
+    name varchar(60),
+    role varchar(40),
+    nat_ins_no INT,
+    bank_account varchar(40),
+    salary DECIMAL(10, 2),
+    PRIMARY KEY (employee_id)
+);
+
+CREATE TABLE `Client` (
+    client_id INT AUTO_INCREMENT,
```

Comment:
where is the primary key?

Reply:
PK constraint is defined at the bottom of the table deffinition